web/public/js/controllers/user-profile-controller.js

```diff
@@ -235,13 +236,14 @@ function cdUserProfileCtrl($scope, $rootScope, $state, $window, auth, cdUsersSer
       alertService.showError( $translate.instant('Error loading Dojos') + ' ' + err);
     });
 
-    cdUsersService.loadDojoAdminsForUserPromise(loggedInUser.data.id).then(
-      function(champions){
-        $scope.loggedInUserIsMemberOfDojoChampion = !!(_.find(champions, function (championForUser) {
-          return championForUser.id === profileUserId;
-        }));
-      }
-    );
+    cdUsersService.loadChildrenForUser($stateParams.userId, function (children) {
```

Comment:
Should these check the user type first? Make sure their an adult before loading children, and a child before loading parents. Nothing major really, just a minor performance improvement by freeing up a connection for a bit. Could also check `$scope.profile.children.length` to see if there's any point fetching.